### PR TITLE
CALL related HUB and STP fixes

### PR DESCRIPTION
**Pull request by OlivierBBB**

### CODE REVIEW

File: hub/instruction_handling/call/generalities/universal.tex [hunk 1]
```diff
@@ -148,31 +148,22 @@
...
-				mxpGas      = \locMxpMemoryExpansionGas ,
+				anchorRow   = i                                 ,
+				relOffset   = \callMiscRowOffset                ,
+				instruction = \locInst                          ,
+				gasHi       = \locGasHi                         ,
+				gasLo       = \locGasLo                         ,
+				valueHi     = \locValueHi                       ,
+				valueLo     = \locValueLo                       ,
+				warmth      = \locCalleeWarmth                  ,
+				exists      = \locCalleeExists \cdot \locIsCall ,
```
Comment (OlivierBBB) [inline hub/instruction_handling/call/generalities/universal.tex:159]: Double mix up + correcting callee existence to only matter for `CALL`.

File: hub/misc/stp/call.tex [hunk 2]
```diff
@@ -9,8 +9,8 @@
 			gasLo       = \locGasLo,
 			valueHi     = \locValueHi,
 			valueLo     = \locValueLo,
-			warmth      = \locTargetExists,
-			exists      = \locTargetWarmth,
+			exists      = \locTargetExists,
+			warmth      = \locTargetWarmth,
```
Comment (OlivierBBB) [inline hub/misc/stp/call.tex:13]: fix: order should be
- exists
- warm

everywhere

File: stp/vanishing.tex [hunk 3]
```diff
@@ -1,30 +1,29 @@
...
+				\valHi _{i} & = & 0 & (\trash) \\
+				\valLo _{i} & = & 0 & (\trash) \\
 			\end{array}\right.
 		\]
+		\saNote{}
+		Recall that neither \inst{DELEGATECALL} nor \inst{STATICCALL} have a ``\texttt{value}'' argument.
+	\item[\underline{Non \inst{CALL}, \inst{CALL}-type instructions:}]
+		\If $\stpCallCode _{i} + \stpDelegateCall _{i} + \stpStaticCall _{i} = 1$
+		\Then we impose that
+		\[ \existence _{i} = 0 \]
```
Comment (OlivierBBB) [inline stp/vanishing.tex:24]: Only `CALL` requires `exists`.

**Review by letypequividelespoubelles — APPROVED** [review]
LGTM

**Pull request merged**